benchmarks/run_naslib.py

```diff
@@ -0,0 +1,103 @@
+import argparse
+import os
+import subprocess
+
+
+def run(args: argparse.Namespace) -> None:
+    kurobako_cmd = os.path.join(args.path_to_kurobako, "kurobako")
+    subprocess.run(f"{kurobako_cmd} --version", shell=True)
+
+    os.makedirs(args.out_dir, exist_ok=True)
+    study_json_filename = os.path.join(args.out_dir, "studies.json")
+    solvers_filename = os.path.join(args.out_dir, "solvers.json")
+    problems_filename = os.path.join(args.out_dir, "problems.json")
+
+    # Ensure all files are empty.
+    for filename in [study_json_filename, solvers_filename, problems_filename]:
+        with open(filename, "w"):
+            pass
+
+    searchspace_datasets = [
+        "nasbench201 cifar10",
+        "nasbench201 cifar100",
+        "nasbench201 ImageNet16-120",
+    ]
+
+    for searchspace_dataset in searchspace_datasets:
+
+        python_command = f"benchmarks/naslib/problem.py {searchspace_dataset}"
+        cmd = (
+            f"{kurobako_cmd} problem command python3 {python_command}"
+            f"| tee -a {problems_filename}"
+        )
+        subprocess.run(cmd, shell=True)
+
+    # Create solvers.
+    sampler_list = args.sampler_list.split()
+    sampler_kwargs_list = args.sampler_kwargs_list.split()
+    pruner_list = args.pruner_list.split()
+    pruner_kwargs_list = args.pruner_kwargs_list.split()
+
+    if len(sampler_list) != len(sampler_kwargs_list):
+        raise ValueError(
+            "The number of samplers does not match the given keyword arguments. \n"
+            f"sampler_list: {sampler_list}, sampler_kwargs_list: {sampler_kwargs_list}."
+        )
```

Comment:
How about testing this for `pruner_list` and `pruner_kwargs_list`?

Reply:
Fixed.